liblangutil/Scanner.cpp

```diff
@@ -313,7 +313,6 @@ size_t Scanner::scanSingleLineDocComment()
 {
 	LiteralScope literal(this, LITERAL_TYPE_COMMENT);
 	size_t endPosition = m_source->position();
```

Comment:
Previously, we called `position()` before `advance()`. If the source ended with `///` we captured `//` only.